documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -28,4 +28,5 @@
 ### Resolved issues
 
 * (IDETECT-4177) - [detect_product_short] no longer requires that the X-Artifactory-Filename header is set when specifying an internally hosted version in [bd_product_long].
+* (IDETECT-3512) - [detect_product_short] now correctly sends an indicator specifying all categories when `detect.project.clone.categories` is set to ALL. This prevents problems where [bd_product_long] and [detect_product_short] disagree on the full list of categories.
```

Comment:
Suggested text tweak:
"To prevent issues when [bd_product_long] and [detect_product_short] disagree on the full list of categories, [detect_product_short] now sends an indicator specifying "all categories" when `detect.project.clone.categories` is set to ALL."